src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -2121,11 +2233,10 @@ export default class GeFormRenderer extends LightningElement{
     }
 
     get hasCapturedPayment() {
-        const paymentStatus = this.getFieldValueFromFormState(
-            apiNameFor(PAYMENT_STATUS)
-        );
-        return paymentStatus &&
-            paymentStatus === this.PAYMENT_TRANSACTION_STATUS_ENUM.CAPTURED;
+        const paymentStatus = this.getFieldValueFromFormState(PAYMENT_STATUS);
+        return paymentStatus
```

Comment:
We could add boolean variables here for CAPTURED and SUBMITTED, and simplify the return statement.